### DISCUSSION TITLE
@rneui/base and @rneui/themed version code is not matched

### DISCUSSION THREAD

**Discussion by lovetingyuan** [react-native-elements/react-native-elements#3870] [General]
@rneui/themed package.json
 
```js
{
  "name": "@rneui/themed",
  "version": "4.0.0-rc.8",
  
  "peerDependencies": {
      "@rneui/base": "4.0.0-rc.7"
    },
    "devDependencies": {
      "@rneui/base": "4.0.0-rc.7"
    },
}
```
the version is not matched.

**Comment by theianmay** [comment]
## 🚀 React Native Elements v5 Beta is Live!  

Hello everyone! 👋 Thank you all for your patience while our new team has been hard at work. 
We’re excited to announce that **v5.0.0-beta.1** is now available! 

This is a **beta release**, intended for testing and feedback. The **full stable v5.0.0** is still in the pipeline.  

---
###⚠️ What you need to know
- This is not yet stable, intended for testing and feedback.
- For example, a known issue is: Button disabled state styling on Android. Workarounds and discussion available here: #3985
- Associated issue: https://github.com/facebook/react-native/issues/53849
- You can install the beta via NPM:
```bash
npm install @rneui/base@beta @rneui/themed@beta
```
---

### 📝 Feedback
Please provide **bugs, suggestions, or feedback**:  
- GitHub Discussion: https://github.com/react-native-elements/react-native-elements/discussions/4007 
- Discord: https://discord.com/invite/e9RBHjkKHa

**Next Steps:**  
- I will be closing this for now to keep things organized.
- All ongoing beta feedback should go to the Discussion or Discord.

**Reply by lovetingyuan** [reply]
wonderful! Will RNE continue to support the latest RN and Expo version?

**Reply by theianmay** [reply]
Sorry for the delay @lovetingyuan , was out for the US holiday, but yes that will be our goal for the future to keep things up to date!
